ui/src/app/machines/views/MachineDetails/MachineTests/MachineTestsDetails/MachineTestsDetails.tsx

```diff
@@ -37,6 +43,16 @@ const MachineTestsDetails = (): JSX.Element | null => {
     }
   }, [dispatch, scriptResults, loading, id]);
 
+  useEffect(() => {
+    if (!logs && result) {
+      ["combined", "stdout", "stderr", "result"].forEach((type) =>
```

Comment:
It would be good to have an enum for these. Maybe `LogType` could be turned into an enum and be used here.